pkg/scheduler/framework/plugins/clusterlocality/cluster_locality.go

```diff
@@ -38,11 +37,6 @@ func (p *ClusterLocality) Score(ctx context.Context,
 		return framework.MinClusterScore, framework.NewResult(framework.Success)
 	}
 
-	replicas := util.GetSumOfReplicas(spec.Clusters)
-	if replicas <= 0 {
-		return framework.MinClusterScore, framework.NewResult(framework.Success)
-	}
```

Comment:
I think removing this check would fix the issue tranced by https://github.com/karmada-io/karmada/pull/3290, right?

Reply:
I suggest letting this PR focus on this issue.

Reply:
> I think removing this check would fix the issue tranced by #3290, right?

Yes, do we have a plan to unify the form whether duplicated is explict or implicit?

Reply:
Yes we can do that, but please clarify the benefits in the next PR and we can discuss it there.
My idea is don't mix too many things in a PR, that will complicate things.